Intersect.Client/Networking/PacketSender.cs

```diff
@@ -41,7 +40,7 @@ public static void SendNeedMap(Guid mapId)
 
         public static void SendMove()
         {
-            Network.SendPacket(new MovePacket(Globals.Me.MapId, Globals.Me.X, Globals.Me.Y, Globals.Me.Dir));
+            Network.SendPacket(new MovePacket(Globals.Me.MapId, Globals.Me.X, Globals.Me.Y, (byte)Globals.Me.Dir));
```

Comment:
I think the cast can be done inside the constructor

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/222616120-cd1042cf-6b03-4ac6-a67d-a6912beafa6c.png)